AK/Complex.h

```diff
@@ -185,7 +185,7 @@ class [[gnu::packed]] Complex {
     }
 
     template<AK::Concepts::Arithmetic U>
-    constexpr Complex<T> operator/(const Complex<U>& a)
+    constexpr Complex<T> operator/(const Complex<U>& a) const
```

Comment:
These const changes probably belong in the next commit, squash mishap perhaps?

Reply:
Not really, the test cases starts to fails because of the `s/constinit/constexpr/` change. So these are required, for the code to compile.
I didn't knew how to deal with the situation, because both commits are very tight coupled... Maybe it should be done in one commit.

Reply:
Alright, please mention that in the commit message then : )